### PR TITLE
Add tests for Language classes

### PR DESCRIPTION
**Pull request by imobachgs**
Extend testing and documentation for Language related classes.

### CODE REVIEW

File: service/test/dinstaller/dbus/language_test.rb [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  describe "#available_languages" do
+    it "returns the list of available languages" do
+      expect(subject.available_languages).to eq(
+        [["en_US", "English (US)", {}], ["de_DE", "Deutsch", {}]]
+      )
+    end
+  end
+
+  describe "#marked_for_install" do
+    it "returns the languages that are selected for installation" do
```
Comment (dgdavid) [inline service/test/dinstaller/dbus/language_test.rb:53]: (stupid) Question: Can be a list of languages marked for installation.

File: service/lib/dinstaller/language.rb [hunk 2]
```diff
@@ -37,15 +37,25 @@ def initialize(logger)
       @languages = []
     end
 
+    # Probes the available system languages
+    #
+    # @param _progress [Progress] Progress reporting object
     def probe(_progress)
```
Comment (dgdavid) [inline service/lib/dinstaller/language.rb:43]: Question: why those methods receive a param that are not used?

**Review by dgdavid — APPROVED** [review]
Looks good, thanks

File: service/test/dinstaller/dbus/language_test.rb (hunk 1 above)
Comment (imobachgs) [reply service/test/dinstaller/dbus/language_test.rb:53]: Not yet, but I think it is planned for the future.

File: service/lib/dinstaller/language.rb (hunk 2 above)
Comment (imobachgs) [reply service/lib/dinstaller/language.rb:43]: For consistency with the rest of `#probe` and `#install` methods.

**Pull request merged**